toolkit/scripts/toolchain/check_manifests.sh

```diff
@@ -8,19 +8,19 @@ TOOLCHAIN_SPEC_LIST_FILE=$1
 SPECS_DIR=$2
 MANIFESTS_DIR=$3
 DIST_TAG=$4
-ARCH=$5
+DISTRO_MACRO=$5
```

Comment:
Are we still using `ARCH`? It looks like we're still passing it as an argument.
```suggestion
DISTRO_MACRO=$5
ARCH=$6
```

Reply:
Good catch, mangled that one during a revert I think.